src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/templates/layouts/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import {FC} from 'react';
+
+export interface HeaderProps {
+    Title: FC;
+    Description: FC;
+    Goal: FC;
```

Comment:
Actually, even better would be:
```tsx
import type {Goal as GoalComponent} from './some-place}

{
    Goal: typeof GoalComponent
}
```

Reply:
This would be ideal except once you specify the typeof in props, typescript is going to expect it's required props...

<img width="964" alt="Screen Shot 2022-11-21 at 10 23 29 AM" src="https://user-images.githubusercontent.com/10138447/203092887-33a7f99d-9857-4130-bde7-549ffc0251f8.png">

Same here if we changed the FieldLabel you created to use typeof:

<img width="968" alt="Screen Shot 2022-11-21 at 10 24 25 AM" src="https://user-images.githubusercontent.com/10138447/203092992-92193053-2b53-46be-82e8-ab42512cea09.png">